exoctk/exoctk_app/templates/limb_darkening.html

```diff
@@ -93,10 +93,10 @@ <h1>Limb Darkening Calculator</h1>
                     <div class="col-sm-10">
                         {% for subfield in form.modeldir %}
                             <div class='radio'>
-                                {% if subfield.label.text == 'Phoenix ACES' %}
-                                    {{ subfield(style='margin-left:0px;', id='aces', checked='checked') }}{{ subfield.label }}
+                                {% if subfield.label.text == 'Kurucz ATLAS9' %}
+                                    {{ subfield(style='margin-left:0px;', id='atlas9', checked='checked') }}{{ subfield.label }}
                                 {% else %}
-                                    {{ subfield(style='margin-left:0px;', id='atlas9') }}{{ subfield.label }}
+                                    {{ subfield(style='margin-left:0px;', id='aces') }}{{ subfield.label }}
```

Comment:
@hover2pi Why switch from ACES --> ATLAS?

Reply:
ATLAS9 is the default model grid but was second in the list of radio buttons, which seemed silly. So I switched them.